### ISSUE TITLE
Remove unchecked type cast from code

### ISSUE THREAD

**Issue by stonezdj** [vmware/harbor#5307]
There are lots of code like that:
```
cfg[common.AdmiralEndpoint].(string)

```
https://tour.golang.org/methods/15

It will cause panic with nil value, should change like that:

```
	if result, ok := cfg[common.LDAPGroupBaseDN].(string); ok {
		ldapGroupConf.LdapGroupBaseDN = result
	}

```

**Comment by reasonerjt** [comment]
Do we need to keep this open?

**Comment by stonezdj** [comment]
Fixed in #5321
